templates/PROJECT_project_working_group_charter_WORKING_GROUP_NAME.md

```diff
@@ -0,0 +1,59 @@
+# Charter of the ROS Project Working Group "[Working Group Name]"
+
+DD Month 2024
+
+### Names
+
+The names of the Working Group shall be "ROS Working Group on [Working Group Name]", "WG on [Working Group Name]" and "ROS-WG-[Working Group acronym]".
+
+
+### Chair
+
+The Chair of the Working Group shall be [Name of PMC Member of the Project who will chair the WG].
+
+
+### Purpose
+
+The Working Group is formed to [purpose of working group].
+
+
+### Tasks
+
+The Working Group shall perform the following tasks.
+
+1. [Task]
+
+1. [Task]
+
+1. [Task]
+
+
+### Expected outputs
+
+The Working Group shall, upon completion of its tasks and as appropriate while performing them, provide to the [Project] PMC the following.
+
+1. [List of expected outputs, one per list item].
+
+[The WG's productivity and progress will be judged by these outputs, and the PMC is allowed to dissolve the WG if it does not produce these outputs as expected.]
+
+
+### Completion date
+
+The Working Group shall complete its work by [expected completion date].
+The Working Group shall be automatically dissolved on that date, unless its Charter is explicitly renewed by the [Project] PMC.
+
+
+### Meetings
+
+The Working Group shall meet at least every [number] weeks.
+Meetings shall be noticed at least three (3) calendar days in advance to the [Project] PMC Members and Committers.
+Meeting notice shall include the date, time, and URL of the online meeting.
+A single announcement noticing a series of meetings together shall be considered sufficient notice for all meetings in the series, provided that notice of the cancellation of one of those meetings shall be given at least one (1) calendar day in advance.
+
+Meetings shall be held online via a suitable teleconference service as specified by the [Project] PMC.
+
+
```

Comment:
```suggestion
For meetings that require community input or feedback, working groups should announce their meetings on [ROS Discourse](https://discourse.ros.org/c/ng-ros/25) at least one week in advance and include their event in the [official ROS events calendar](https://calendar.google.com/calendar/u/0/embed?src=agf3kajirket8khktupm9go748@group.calendar.google.com&ctz=Etc%2FUTC).
```

Reply:
This is a template, not a guide, so we should write it as:

> For a meeting that requires community input or feedback, the Working Group shall announce the meeting one week in advance in [public announcement channel, e.g. [ROS Discourse](https://discourse.ros.org)], and record the event in the [official OSRF events calendar](https://calendar.google.com/calendar/u/0/embed?src=agf3kajirket8khktupm9go748@group.calendar.google.com&ctz=Etc%2FUTC).